snapcraft/plugins/catkin.py

```diff
@@ -245,12 +246,18 @@ def build(self):
 
         self.run(
             ['rm', '-f',
-             'opt/ros/' + self.options.rosversion + '/.catkin',
-             'opt/ros/' + self.options.rosversion + '/.rosinstall',
-             'opt/ros/' + self.options.rosversion + '/setup.sh',
-             'opt/ros/' + self.options.rosversion + '/_setup_util.py'],
+             'opt/ros/' + self.options.rosversion + '/.rosinstall'],
```

Comment:
This deserves an integration test, to check that the files you want end up in the right dirs.
Feel free to add that in a later branch, because that's debt from before.